scripts/esmfold_inference.py

```diff
@@ -121,7 +127,15 @@ def create_batched_sequence_datasest(
     logger.info(f"Loaded {len(all_sequences)} sequences from {args.fasta}")
 
     logger.info("Loading model")
+
+    # Use pre-downloaded ESM weights from model_pth.
+    if args.model_pth is not None and os.path.exists(args.model_pth) and 'checkpoints' in os.listdir(args.model_pth):
```

Comment:
This leads to unexpected behavior if you have a typo in your `model_dir`. Better to fail with an explicit error rather than quietly.
```suggestion
    if args.model_dir is not None:
```

Also we'd want people to run with custom torch.hub dir even if it still has to be populated by using the model for the first time.

Reply:
Yes that would be quite useful!